pupil-spa/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
As we are using typescript we will be using tslint - rules are defined in `tslint.json` - which we can tweak.

I don't think there is any harm in leaving it in, and perhaps much good.